packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -204,8 +228,12 @@ export default class PciStoragesContainersService {
       );
   }
 
-  getContainerUrl(projectId, container, file = null) {
-    return this.getAccessAndToken(projectId).then(({ endpoints }) => {
+  getContainerUrl(projectId, container, file = null, s3StorageType) {
```

Comment:
**nitpick**
```suggestion
  getContainerUrl(projectId, container, s3StorageType, file = null) {
```
Optional parameters should always be at the end

Reply:
Hello Jeremy, s3StorageType is also optional parameter.

Reply:
Sorry I mean default parameter, it is good practice to set them at the end. See https://eslint.org/docs/latest/rules/default-param-last